capella_console_client/tasking.py

```diff
@@ -1,13 +1,133 @@
-from typing import Optional, Dict, Any, List
-from datetime import datetime
+from typing import Optional, Dict, Any, List, Union
+from datetime import datetime, timedelta
 
-import dateutil.parser
+import geojson
+from dateutil.parser import parse, ParserError
 
 from capella_console_client.session import CapellaConsoleSession
-from capella_console_client.validate import _validate_uuid
+from capella_console_client.validate import _validate_uuid, _snake_to_camel
 from capella_console_client.config import TASKING_REQUEST_DEFAULT_PAGE_SIZE
 from capella_console_client.logconf import logger
-from capella_console_client.enumerations import TaskingRequestStatus
+from capella_console_client.enumerations import (
+    TaskingRequestStatus,
+    ObservationDirection,
+    OrbitState,
+    ProductType,
+    ProductClass,
+    OrbitalPlane,
+    CollectionTier,
+    InstrumentMode,
+    Polarization,
+    ArchiveHoldback,
+)
+
+COLLECT_CONSTRAINTS_KEYS = set(
```

Comment:
I'd rename to TASKING_REQUEST_COLLECT_CONSTRAINTS_KEYS plus make this a frozenset

Probably not worth it to share the common keys out between TR and RR.

Reply:
Also looking at the code base I decided that it would make sense to centralize these collections of identifiers within `capella_console_client/config.py` - so if possible let's move it there

Reply:
Done, good idea to move it there.